src/data/posts/twitter-and-the-perils-of-obedience.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: Twitter and the Perils of Obedience
+excerpt: 'Reflections on the current state of Twitter, and its similarity to the shocking results of a 1961 experiment by Yale Researcher Stanley Milgram.'
+tags: [philosophy, twitter, vote, psychology]
+date: 11-28-22
+published: true
+path: twitter-and-the-perils-of-obedience
+---
+
+## Milgram's Experiment
+
+In 1961, a Yale University researcher named Stanley Milgram conducted a series of experiments as a part of his research on behavioral psychology. The goal of the experiment was to test participants' tendency to obey authority figures when under pressure.
+
+Throughout the course of the experiment, the experiment was conducted on 40 volunteers; all men between the ages of 20 and 50, with a variety of education and economic backgrounds.
+
+Milgram later summarized findings from the experiment in an essay titled "The Perils of Obedience". The results were extremely unexpected.
+
+### The Experiment: Setup
+
+Participants who were asked to take part in a study on learning and memory. Each study was led guided by an _Experimenter_ dressed in a lab coat, who explained to participants that they'd be doing a "scientific study on memory and learning". When they arrived, participants were paired with an actor who they were _told_ was another participant in the study.
+
+The actor was secretly working with the researcher, and during the experiment was instructed to play the role of a _Learner_.
+
+The study participant was given the role of _Teacher_. The experimenter then explained that the study would involve the Teacher asking the Learner series of questions. For each incorrect answer, the Teacher would be asked to hit a button to administer an electric shock to the actor. Subsequent incorrect answers would result in a shock of increasing intensity, from 15V to 450V.
```

Comment:
Not immediate clear who the participant was, maybe "The volunteer study participant" or something similar?

Reply:
.... that the study would involve the Teacher asking the Learner **a** series of questions.

Reply:
For each incorrect answer, the Teacher would be asked to hit a button <s>to</s> **that would** administer an electric shock to the actor. Subsequent incorrect answers would result in a shock of increasing intensity, from 15V to 450V.